### PR TITLE
chore(release): 0.10.0

### PR DESCRIPTION
**Pull request by seigler**
* Revise changelog
* Increment version in spark `package.json`
* tag v0.10.0

### CODE REVIEW

**Review by WilliamKelley — CHANGES_REQUESTED** [review]
Need to modify the root changelog too

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ See project specific changelogs for details
 - [`@prenda/spark` here](https://github.com/prenda-school/prenda-spark/blob/main/libs/spark/CHANGELOG.md)
 - [`@prenda/spark-icons` here](https://github.com/prenda-school/prenda-spark/blob/main/libs/spark-icons/CHANGELOG.md)
 
+# [0.10.0](https://github.com/prenda-school/prenda-spark/compare/v0.9.0...v0.10.0) (2021-07-29)
```
Comment (WilliamKelley) [inline CHANGELOG.md:10]:
```suggestion
# [0.10.0](https://github.com/prenda-school/prenda-spark/compare/v0.9.0...v0.10.0) (2021-08-06)
```

File: libs/spark/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
 
 This file was generated using [@jscutlery/semver](https://github.com/jscutlery/semver).
 
+# [0.10.0](https://github.com/prenda-school/prenda-spark/compare/v0.9.0...v0.10.0) (2021-08-05)
```
Comment (WilliamKelley) [inline libs/spark/CHANGELOG.md:5]:
```suggestion
# [0.10.0](https://github.com/prenda-school/prenda-spark/compare/v0.9.0...v0.10.0) (2021-08-06)
```

**Review by WilliamKelley — CHANGES_REQUESTED** [review]

**Review by WilliamKelley — CHANGES_REQUESTED** [review]
Bump spark-icons version and peer dep of spark. Add a "No changes" changelog entry.

Bump the root workspace version as well.

**Review by WilliamKelley — APPROVED** [review]

**Review by slingner — APPROVED** [review]

**Pull request merged**